src/foundry/common/documents.mjs/baseAmbientLight.d.ts

```diff
@@ -1,12 +1,15 @@
+import { ConfiguredDocumentClass } from '../../../types/helperTypes';
 import { DocumentMetadata } from '../abstract/document.mjs';
 import { Document } from '../abstract/module.mjs';
 import * as data from '../data/data.mjs';
-import { BaseScene } from './baseScene';
 
 /**
  * The AmbientLight embedded document model.
  */
-export declare class BaseAmbientLight extends Document<data.AmbientLightData, BaseScene> {
+export declare class BaseAmbientLight extends Document<
+  data.AmbientLightData,
+  InstanceType<ConfiguredDocumentClass<typeof Scene>>
```

Comment:
I'd prefer if we only directly reference the `Base...` classes here. I know, in the end, it doesn't really make a difference but I think it's more consistent. After all, in foundry itself, the client side classes cannot be referenced here because this code is also used on the server side.

Reply:
https://github.com/League-of-Foundry-Developers/foundry-vtt-types/pull/837#discussion_r650503873 :grin: 

(But I do also prefer the BaseScene)